extensions/stats/plugin.cc

```diff
@@ -377,28 +413,25 @@ void PluginRootContext::initializeDimensions() {
   build.record(
       1, "proxy",
       absl::StrCat(flatbuffers::GetString(local_node.istio_version()), ";"));
-}
+  return true;
+}  // namespace Stats
```

Comment:
nit: seems to be wrong comment. This is not name of Stats namespace

Reply:
indeed, it's wrong.